src/UseCases/ApplyForMembership/ApplyForMembershipRequest.php

```diff
@@ -34,11 +35,14 @@ class ApplyForMembershipRequest {
 	private string $applicantPhoneNumber;
 	private string $applicantDateOfBirth;
 
+	// TODO throw them out:
```

Comment:
You can get some support from your IDE by adding docblocks with `@deprecated` on the accessor methods and properties.